.github/workflows/CI.yml

```diff
@@ -16,7 +16,7 @@ jobs:
           - stable
           - beta
           - nightly
-          - "1.40.0"
+          - "1.42.0"
```

Comment:
1.48 is due tomorrow, so maybe let's just go with that tomorrow instead of updating the CI twice :)

Reply:
That happened now

@bilelmoussaoui can you update this one?

Reply:
Done